.pipelines/pipeline.user.windows.yml

```diff
@@ -120,6 +120,12 @@ build:
             - 'modules\launcher\Plugins\Microsoft.Plugin.WindowWalker\Wox.Infrastructure.dll'
             - 'modules\launcher\Plugins\Microsoft.Plugin.WindowWalker\Wox.Plugin.dll'
             - 'modules\launcher\Plugins\Microsoft.Plugin.WindowWalker\Telemetry.dll'
+            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\Microsoft.Plugin.Service.dll'
+            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\Wox.Plugin.dll'
+            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\PowerToysInterop.dll'
+            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\Microsoft.PowerToys.Settings.UI.Lib.dll'
+            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\ManagedTelemetry.dll'
+            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\ManagedCommon.dll'
```

Comment:
I understand they were already used before this PR, but `Microsoft.Plugin` is not a good prefix, it should be `Microsoft.PowerToys.Run.Plugin.`
And we should also stop using `Wox`.
We should consider fixing that.

Reply:
Related to https://github.com/microsoft/PowerToys/issues/2125 and https://github.com/microsoft/PowerToys/issues/2055
Should I use new names for this plugin and have different names in 0.31?

Reply:
When adding new binaries, let's use the new naming convention, for the old one we will gradually update them in small PRs.

Reply:
@enricogior `Microsoft.PowerToys.Run.Plugin` or `Microsoft.PowerToys.Run.Plugins`?

Reply:
`Microsoft.PowerToys.Run.Plugin`